gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/GremlinClientTests.cs

```diff
@@ -68,6 +69,20 @@ public async Task ShouldHandleBigResponse()
             }
         }
 
+        [Fact]
+        public async Task ShouldHandle_JToken_Request()
+        {
+            var gremlinServer = new GremlinServer(TestHost, TestPort);
+            using (var gremlinClient = new GremlinClient(gremlinServer))
+            {
+                var gremlinScript = "'someString'";
+                
+                var response = await gremlinClient.SubmitWithSingleResultAsync<JToken>(gremlinScript);
```

Comment:
I'm not sure but wouldn't this test also succeed without your change as the returned data is just a string here? Maybe it would make more sense to return something that includes a type in its GraphSON form like an `int`?

Reply:
It would not anymore since there's more assertions now about the actual JObject that's returned, which is not just a plain string but

    {
        "@type": "g:List",
        "@value": [
            "someString"
         ]
    }

Reply:
Hmm, didn't realize that the result would be wrapped in a list in cases like this, but you obviously proved that it's the case with the added asserts.